elements/pfe-tabs/src/pfe-tabs.js

```diff
@@ -303,6 +308,8 @@ class PfeTabs extends PFElement {
   }
 
   _panelForTab(tab) {
+    if (!tab || !tab.controls) return;
```

Comment:
Don't run this query if the tab or the tab controls information doesn't exist (yet)